packages/lesswrong/components/form-components/SectionFooterCheckbox.tsx

```diff
@@ -19,7 +19,7 @@ const styles = (theme: ThemeType) => ({
   },
   checkbox: {
     color: theme.palette.lwTertiary.main,
-    padding: "2px 8px 0 0",
+    padding: "1px 8px 0 0",
```

Comment:
it sure looks like the 2px padding was just... wrong (looked visibly off), and not sure why it was every a good idea.

Reply:
You previously changed it from 1px to 2px in https://github.com/ForumMagnum/ForumMagnum/commit/7ad9ad80330720bf5a7bec7477a33ec6ce2b00d9 and didn't say why